### DISCUSSION TITLE
create sequelize model with plain js object

### DISCUSSION THREAD

**Discussion by muhammadumerfarooq** [sequelize/sequelize#14144] [Help & Questions] [answered]
Hi, 
Is it possible to define sequelize model from plain js object ?

**Comment by ephys** [comment, marked as answer]
There are only two ways to define a Model using base sequelize, declaring a class that extends `Model`, or using `sequelize.define` (https://sequelize.org/v6/manual/model-basics.html)

Edit: In Sequelize 7 (or if you use Sequelize-TypeScript), you can also use decorators
